File: 2 - Condições/Java/condicoes11.java
//Escreva um programa que leia dois nomes de times de futebol, a quantidade de gols do primeiro time, a quantidade de gols do segundo time e mostre o nome do time vencedor da partida.

import java.util.Scanner;

public class condicoes11 {

  public static void main(String[] args) {
    System.out.println("Digite o nome dos dois times: ");
    Scanner scan = new Scanner(System.in);
    String time1 = scan.nextLine();
    String time2 = scan.nextLine();
    System.out.println("Digite a quantidade  dos gols de cada time: ");
    int gol1 = scan.nextInt();
    int gol2 = scan.nextInt();
    if (gol1 > gol2) {
      System.out.println(time1 + " venceu!");
    } else if (gol1 == gol2) {
      System.out.println("Deu empate!");
    } else {
      System.out.println(time2 + " venceu!");
    }
  }
}

File: 2 - Condições/Ruby/condicoes11.rb
#Escreva um programa que leia dois nomes de times de futebol, a quantidade de gols do primeiro time, a quantidade de gols do segundo time e mostre o nome do time vencedor da partida.

puts "Escreva o nome do primeiro time: "
nome1 = gets.chomp
puts "Escreva o número de gols do primeiro time: "
gols1 = gets.to_i
puts "Escreva o nome do segundo time: "
nome2 = gets.chomp
puts "Escreva o número de gols do segundo time: "
gols2 = gets.to_i

if(gols1 > gols2)
	puts "O time vencedor é o: #{nome1}"
elsif (gols1 == gols2)
	puts "Deu empate"
else
	puts "O time vencedor é o: #{nome2}"
end
